staging_service/import_specifications/individual_parsers.py

```diff
@@ -316,3 +317,50 @@ def parse_excel(path: Path) -> ParseResults:
         return ParseResults(frozendict(results))
     else:
         return _error(Error(ErrorType.PARSE_FAIL, "No non-header data in file", spcsrc))
+
+
+def parse_dts_manifest(path: Path) -> ParseResults:
+    """
+    Parse the provided DTS manifest file. Expected to be JSON, and will fail otherwise.
+    The manifest should have roughly this format, with expected keys included:
```

Comment:
If I provide a manifest that only has roughly the format below will it parse? I'd expect not

Reply:
In this PR? No, it's mostly a null placeholder to satisfy the mapping because I didn't want to make this one much larger. Basic fails are trapped properly, and a "passing" file returns nothing.

Reply:
I'm talking about this line

> The manifest should have roughly this format

Which seems to imply that the parser is smart enough to deal with variations in format, which I'd assume is something you're not interested in doing

Reply:
Updated that docstring to be more clear.

Reply:
Did you push the commit? I'm not seeing any changes

Reply:
:+1: